src/runtime/bootstrap-custom-element.ts

```diff
@@ -36,6 +43,25 @@ export const proxyCustomElement = (Cstr: any, compactMeta: d.ComponentRuntimeMet
     cmpMeta.$flags$ |= CMP_FLAGS.needsShadowDomShim;
   }
 
+  // TODO(STENCIL-914): this check and `else` block can go away and be replaced by just `BUILD.scoped` once we
+  // default our pseudo-slot behavior
+  if (BUILD.patchPseudoShadowDom && BUILD.scoped) {
```

Comment:
I pulled over this `if-else` block directly from `bootstrapLazy`. Without it, we don't patch anything for scoped behavior when using a custom elements build.

Reply:
I'd be surprised if that was the case. As a developer, I'd expect to have (mostly) the same runtime behavior for lazy and non-lazy bundles (with the obvious exceptions for how they load). Maybe the use-case isn't there for it, but seems like we should at least offer parity. If we offer `scoped` on custom elements builds, I'd think the down-stream features of that choice should be valid.

Reply:
yeah that makes sense to me 👍